gatsby/whatisgatsby.rst

```diff
@@ -22,4 +21,4 @@ with modern best-practices and tools:
 
 GatsbyJS sets up a project using the latest technologies which pulls data from almost any source at all and generates a static site, just by whipping together a couple of Gatsby plugins together or directly with all configuration by using a starter.
```

Comment:
Sentence run-on. Suggest:

```
GatsbyJS sets up a project using the latest technologies, pulling data from almost any source, and generating a static site.
It can be done either by whipping together a couple of GatsbyJS plugins, or by using a starter project with all its default configuration.
```